package/libs/openssl/patches/430-e_devcrypto-make-the-dev-crypto-engine-dynamic.patch

```diff
@@ -112,117 +106,2462 @@ index 1db771971c..33a25d7004 100644
  
      ENGINES_NO_INST=ossltest dasync
      SOURCE[dasync]=e_dasync.c
-diff --git a/crypto/engine/eng_devcrypto.c b/engines/e_devcrypto.c
-similarity index 95%
```

Comment:
Here some magic happens.

Reply:
Indeed, our "refresh" does not reproduce renames in patches and thus will convert this into "delete" and "add".
I had a similar situation recently with a kernel bump:
https://github.com/openwrt/openwrt/commit/a45953c2f2c8c0c488eca13c2b8d9164a71ed5dc

Reply:
Is that something we should fix or is that excepted? @nbd168 thoughts?

Reply:
Well, remove/add will break if there is a change to the original file, so get the developer's attention in this case. However, this will only help if the developer does not forget to also fix the add part.
With rename in patch files, changes in the original file will be transferred to the new file as well. So, this is safer for rebasing, but you won't be "notified" by something breaking apart (since just taking the change into the new file might as well be bad).

So, essentially, the add/remove syntax contains more actual information and could thus be considered more robust, although it's actual effectiveness depends on the "user".